src/assets/Utils.ts

```diff
@@ -171,6 +174,27 @@ const hexToRgbArray = (hex: string) => {
   }
 }
 
+const sortDocumentName = (docArray: DocumentName[]) => {
```

Comment:
We can name this function in a more generic way, as it can be used for sorting anything with names

```suggestion
const sortByName = (dataArray: GenericObject[]) => {
```

or maybe 

```suggestion
const sortArrayByName = (dataArray: GenericObject[]) => {
```